config/default/contact-summary-extras.js

```diff
@@ -0,0 +1,445 @@
+const moment = require('moment');
+const today = moment().startOf('day');
+
+const isReportValid = function (report) {
+  if (report.form && report.fields && report.reported_date) return true;
+  return false;
+};
+//
```

Comment:
There are a few unnecsesary comments around like this one.

Reply:
Thanks, I have removed these meaningless comments.